### PR TITLE
chore(consensus): [CON-1341] inline `node_registration_test` and `ssh_access_to_nodes_test` system tests

### PR DESCRIPTION
**Pull request by kpop-dfinity**

### CODE REVIEW

File: rs/tests/consensus/orchestrator/node_registration_test.rs [hunk 1]
```diff
@@ -1,10 +1,138 @@
...
+    submit_external_proposal_with_test_id, vote_execute_proposal_assert_executed,
+};
 use ic_system_test_driver::systest;
-use ic_tests::orchestrator::node_registration_test::{config, test};
+use ic_system_test_driver::util::{block_on, runtime_from_url};
+use registry_canister::mutations::node_management::do_remove_nodes::RemoveNodesPayload;
+use slog::info;
+use std::str::FromStr;
+
+fn config(env: TestEnv) {
```
Comment (basvandijk) [inline rs/tests/consensus/orchestrator/node_registration_test.rs:34]: What about calling this `setup`? The `config` naming is deprecated.
```suggestion
fn setup(env: TestEnv) {
```

File: rs/tests/consensus/orchestrator/ssh_access_to_nodes_test.rs [hunk 2]
```diff
@@ -1,15 +1,354 @@
...
+        test_env_api::{
+            HasPublicApiUrl, HasTopologySnapshot, IcNodeSnapshot, NnsInstallationBuilder,
+            SubnetSnapshot, TopologySnapshot,
+        },
+    },
+    util::{block_on, get_app_subnet_and_node, get_nns_node},
 };
+use std::net::IpAddr;
+
+pub fn config(env: TestEnv) {
```
Comment (basvandijk) [inline rs/tests/consensus/orchestrator/ssh_access_to_nodes_test.rs:38]:
```suggestion
pub fn setup(env: TestEnv) {
```

**Review by basvandijk — APPROVED** [review]
Awesome job inlining these tests. I'll use you as a shining example that other teams should follow!

File: rs/tests/consensus/orchestrator/node_registration_test.rs (hunk 1 above)
Comment (kpop-dfinity) [reply rs/tests/consensus/orchestrator/node_registration_test.rs:34]: Done, thanks

File: rs/tests/consensus/orchestrator/ssh_access_to_nodes_test.rs (hunk 2 above)
Comment (kpop-dfinity) [reply rs/tests/consensus/orchestrator/ssh_access_to_nodes_test.rs]: Thanks!

**Pull request merged**